### DISCUSSION TITLE
Any reference paper for the psm tool?

### DISCUSSION THREAD

**Discussion by Pzy-ss** [The-OpenROAD-Project/OpenROAD#3395] [Q&A] [answered]
Did the psm module refer to any academic research? If so, it would be better to cite the paper for us to learn the ir-solver and make some improvement. If not, any recommendation for freshmen to understand the whole ir-drop thing? Appreciate it!

**Comment by vijayank88** [comment]
@Pzy-ss 
Have you referred [this](https://github.com/The-OpenROAD-Project/OpenROAD/tree/master/src/psm)?
https://theopenroadproject.org/an-ml-based-iccad-contest-for-static-ir-drop-estimation/

**Comment by maliberty** [comment]
@VidyaChhabria is the author - would you comment.

**Comment by maliberty** [comment, marked as answer]
@VidyaChhabria provided the doc in #3405

**Reply by Pzy-ss** [reply]
Gotta it. Much obliged!

**Reply by Pzy-ss** [reply]
Great paper! I have learned a lot. By the way how can I index the citations in the doc?

**Reply by maliberty** [reply]
@VidyaChhabria would you provide the citations

**Reply by vijayank88** [reply]
@Pzy-ss Are you looking this?
https://github.com/The-OpenROAD-Project/OpenROAD/tree/master/src/psm#cite-this-work

**Reply by Pzy-ss** [reply]
@vijayank88  Thanks! I mean the citations in [https://github.com/The-OpenROAD-Project/OpenROAD/blob/master/src/psm/doc/PDNSim-documentation.pdf]. It seems this paper in the pdf file is one of the chapters in a reference doc?

**Comment by VidyaChhabria** [comment]
Look at PR: https://github.com/The-OpenROAD-Project/OpenROAD/pull/3428 should update the document to have the citations.
